test/Tool/range13.opt

```diff
@@ -0,0 +1,10 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: %FileCheck %s < %t
+
+; CHECK: LGTM
```

Comment:
this is a good test, but in some cases (like this one) you should also have a negative test where you make sure that an optimzation is not performed when it isn't justified by the ranges
even better would be if the positive and negative test cases are right "next to" each other -- that is, there's a corner case right near INT_MAX / 2 where overflow changes from impossible to possible. if your tests straddle that edge case, it is a better way to make sure you got everything right